src/lib/Behat/PageElement/Dialog.php

```diff
@@ -14,7 +14,7 @@ class Dialog extends Element
     /** @var string Name by which Element is recognised */
     public const ELEMENT_NAME = 'Dialog';
 
-    public $dialogContainer;
+    //public $dialogContainer;
```

Comment:
1) Please remove
2) Did the approach with `.show` failed in case of Dialog? I think we should be using it wherever we can, because it's easier to use in development (one selector less to think about when creating Dialog object).

Reply:
It appears that this change was lost in space, time and git commits - works fine with show and added it one more time.